picard/ui/theme.py

```diff
@@ -33,6 +36,60 @@
 )
 
 
+OS_SUPPORTS_THEMES = True
+if IS_MACOS:
+    def is_dark_theme_supported():
+        import platform
+        try:
+            current_version = tuple(map(int, platform.mac_ver()[0].split(".")))[:2]
+        except ValueError:
+            log.warning("Error while converting the MacOS version string into a tuple: %s" % platform.mac_ver()[0])
+            return False
+
+        mojave_version = (10, 14)  # Dark theme support was introduced in Mojave
+        return current_version >= mojave_version
+    OS_SUPPORTS_THEMES = is_dark_theme_supported()
+
+elif IS_HAIKU:
+    OS_SUPPORTS_THEMES = False
+
+
+# Those are values stored in config file:
+class UiTheme(Enum):
+    DEFAULT = 'default'
+    DARK = 'dark'
+    LIGHT = 'light'
+    SYSTEM = 'system'
+
+    def __str__(self):
+        return self.value
+
+    @classmethod
+    def __missing__(cls, value):
```

Comment:
It should be `_missing_` (with single underscores).

Reply:
Good catch, I missed it.